components/SVGWebView.tsx

```diff
@@ -183,40 +178,28 @@ export const SVGWebView = ({ navigation }) => {
               })
             }}
           >
-            <SVGMapUL navigateToShopId={navigateToShopId} />
-          </Animated.View>
-          <Animated.View
-            style={[
-              styles.upButton,
-              { opacity: 0.99, zIndex: 10 },
-              { transform: [{ scale: pulse }] }
-            ]}
-          >
-            <MaterialIcons
-              name="chevron-right"
-              style={{
-                transform: [
-                  { rotate: position === "left" ? "90deg" : "270deg" }
-                ]
-              }}
-              size={40}
-              color="white"
-              onPress={scroll}
+            <SVGMapUL
+              navigateToShopId={navigateToShopId}
+              highlightedShops={highlightedShops[LEVELS.UL]}
             />
           </Animated.View>
+          <PulseButton shouldPulse={shouldPulse} position={position} onPress={scroll}/>
         </View>
+      </ScrollView>
+
+      <View>
+        <Button
+          style={[theme.groupButton, styles.footer]}
+          onPress={() => {

```

Comment:
in places where you call a function with no arguments (or the arguments match what is provided) then you don't need to create another function.
e.g.
`onPress={navigateToShopList}`